kalite/distributed/static/js/distributed/inline/models.js

```diff
@@ -0,0 +1,9 @@
+// Model for the inline tutorial system that fetches from tastypie resource "NarrativeResource"
+
+var NarrativeModel = Backbone.Model.extend({
+    // urlRoot since we are using a model outside of a backbone collection,
+    // enables the url function to generate URLs based on the model id.
+    urlRoot: function() {
+        return '/api/inline/narrative';
```

Comment:
@rtibbles has an open PR to expose django urls in js files by their name (#3556). Once that's merged we should use it here, in case the url changes down the line. More robust that way!